File: app/src/main/java/ru/roscha_akademii/medialib/main/view/MainView.java
package ru.roscha_akademii.medialib.main.view;

import com.hannesdorfmann.mosby.mvp.MvpView;

import java.util.List;

import ru.roscha_akademii.medialib.net.model.Video;

public interface MainView extends MvpView {
    void showHelloToast();

    void showVideoList(List<Video> list);
}

File: app/src/main/java/ru/roscha_akademii/medialib/main/view/MainView.kt
package ru.roscha_akademii.medialib.main.view

import com.hannesdorfmann.mosby.mvp.MvpView

import ru.roscha_akademii.medialib.net.model.Video

interface MainView : MvpView {
    fun showHelloToast()

    fun showVideoList(list: List<Video>)
}
